python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -324,6 +324,40 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
         target=lsst.meas.algorithms.SubtractBackgroundTask,
         doc="Task to perform final background subtraction, just before photoCal.",
     )
+    star_background_max_pedestal = pexConfig.Field(
+        dtype=float,
+        default=0.002,
+        doc="Maximum pedestal allowable in star_background measurement, represented here "
+            "as a fraction of the initial median background already subtracted. If the "
+            "computed value exceeds this value, the background pedestal will be set "
+            "to this value.",
+    )
+    star_background_min_pedestal = pexConfig.Field(
+        dtype=float,
+        default=0.05,
+        doc="Minimum pedestal allowable in star_background measurement, represented here "
+            "as a fraction (the negative of the absolute) of the initial median background "
+            "already subtracted. If the computed value is smaller than this value, the "
+            "background pedestal will be se to this value.",
```

Comment:
se -> set

Reply:
These are gone (for now, at least...will follow up after the next DRP to see if failsafe's are actually needed).